lib/quill_delta.dart

```diff
@@ -463,6 +486,59 @@ class Delta {
     return result;
   }
 
+  /// invert this delta base on baseDelta
+  Delta invert(Delta base) {
+    Delta inverted = new Delta();
```

Comment:
nit: Can replace type annotation with `var` or `final` (even better). Similarly below for `sliceDelta`.